package.json

```diff
@@ -496,122 +496,122 @@
             {
                 "command": "calva.loadFile",
-                "title": "Load (evaluate) current file and its dependencies",
+                "title": "Load Current File and its Dependencies",
                 "category": "Calva"
             },
             {
                 "command": "calva.requireREPLUtilities",
-                "title": "Require REPL utilities, like (doc) etcetera, into current namespace",
+                "title": "Require REPL utilities, like (doc) etcetera, into Current Namespace",
                 "category": "Calva"
             },
             {
                 "command": "calva.lintFile",
-                "title": "Lint current file",
+                "title": "Lint Current File",
                 "category": "Calva"
             },
             {
                 "command": "calva.runNamespaceTests",
-                "title": "Run tests for current namespace",
+                "title": "Run Tests for Current Namespace",
                 "category": "Calva"
             },
             {
                 "command": "calva.runAllTests",
-                "title": "Run all tests",
+                "title": "Run All Tests",
                 "category": "Calva"
             },
             {
                 "command": "calva.rerunTests",
-                "title": "Run previously failing tests again",
+                "title": "Run Failing Tests again",
                 "category": "Calva"
             },
             {
                 "command": "calva.runTestUnderCursor",
-                "title": "Run current test",
+                "title": "Run Current Test",
                 "category": "Calva"
             },
             {
                 "command": "calva.loadNamespace",
-                "title": "Load current namespace in REPL window",
+                "title": "Load Current Namespace in REPL Window",
                 "category": "Calva"
             },
             {
                 "command": "calva.setREPLNamespace",
-                "title": "Switch namespace in current window REPL to namespace of this file",
+                "title": "Switch Namespace in Current REPL Window to Current Namespace",
                 "category": "Calva"
             },
             {
                 "command": "calva.evalCurrentFormInREPLWindow",
-                "title": "Send current form to REPL window and evaluate it",
+                "title": "Evaluate Current Form in REPL Window",
                 "category": "Calva"
             },
             {
                 "command": "calva.evalCurrentTopLevelFormInREPLWindow",
-                "title": "Send current top level form to REPL window and evaluate it (aka defun)",
+                "title": "Evaluate Top Level Form in REPL Window",
                 "category": "Calva"
             },
             {
                 "command": "calva.switchCljsBuild",
-                "title": "Switch CLJS build",
+                "title": "Switch CLJS Build",
```

Comment:
This switches which CLJS build Calva is connected to. Maybe ”Select CLJS Build Connection”...